src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -217,6 +217,119 @@ .maxstack 8
             Assert.Equal(0, accessorBindingData.NumberOfPerformedAccessorBinding);
         }
 
+        [Fact]
+        public void TestPropertyNotAssignedInStructConstructor()
+        {
+            var comp = CreateCompilation(@"
+var x = new C();
+x.P = 10;
+x = new C();
+
+public struct C
+{
+    public C()
+    {
+        System.Console.WriteLine(""In C..ctor: "" + P);
+    }
+
+    public int P { get => field; set => field = value; }
+}
+", options: TestOptions.ReleaseExe.WithSpecificDiagnosticOptions(ReportStructInitializationWarnings));
+            var accessorBindingData = new SourcePropertySymbolBase.AccessorBindingData();
+            comp.TestOnlyCompilationData = accessorBindingData;
+            comp.VerifyDiagnostics(
```

Comment:
>VerifyDiagnostics [](http://example.com/codeflow?start=17&length=17)

Consider calling this on result of ```CompileAndVerify``` below instead.

Reply:
#Pending I'll update this test and other tests too

Reply:
Fixed.